webapp/actions/websocket_actions.jsx

```diff
@@ -237,13 +245,18 @@ function handleLeaveTeamEvent(msg) {
 
         // if they are on the team being removed redirect them to the root
         if (TeamStore.getCurrentId() === msg.data.team_id) {
+            BrowserStore.removeGlobalItem('team');
```

Comment:
What happens if you're removed from a team when you're offline? The next time you log in, will you get stuck in an error state since it's trying to switch to a team that you don't have access to?